.github/workflows/npm-test.yml

```diff
@@ -5,20 +5,25 @@ on: [push]
 jobs:
   build:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [10, 12, 14]
+    name: node-${{ matrix.node }}
     steps:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v1
         with:
-          node-version: 12
+          node-version: ${{ matrix.node }}
       - run: npm ci
       - run: npm test
       - name: Publish Unit Test Results
         uses: EnricoMi/publish-unit-test-result-action@v1.6
-        if: always()
+        if: matrix.node == 12
```

Comment:
does this mean the tests always use node 12?

Reply:
No this just means that I only publish the unit test results (and coverage reports) for node 12. They should be identical across node versions but if you upload them all, sometimes the publish fails because it is too many requests at once